src/print.rs

```diff
@@ -75,8 +74,10 @@ impl LateLintPass for PrintLint {
 fn is_in_debug_impl(cx: &LateContext, expr: &Expr) -> bool {
     let map = &cx.tcx.map;
 
-    if let Some(NodeImplItem(item)) = map.find(map.get_parent(expr.id)) { // `fmt` method
-        if let Some(NodeItem(item)) = map.find(map.get_parent(item.id)) { // `Debug` impl
+    if let Some(NodeImplItem(item)) = map.find(map.get_parent(expr.id)) {
+        // `fmt` method
+        if let Some(NodeItem(item)) = map.find(map.get_parent(item.id)) {
+            // `Debug` impl
             if let ItemImpl(_, _, _, Some(ref tr), _, _) = item.node {
```

Comment:
The comment is now at the wrong place, IMO it should be on the same line as the  corresponding `if let` or on the previous line.